BorrowBookUi.java

```diff
@@ -0,0 +1,98 @@
+import java.util.Scanner;
+
+public class BorrowBookUi {

```

Comment:
BorrowBookUi does not follow the correct coding standards ui is an acronym for user interface
there for it should be BorrowBookUI

Reply:
Okay